### ISSUE TITLE
ANDROID_HOME.../platforms/null/android.jar does not exist

### ISSUE THREAD

**Issue by klutzer** [evant/gradle-retrolambda#90]
I have some problems when building an fxmobile application, The android version appears null..
ANDROID_HOME.../platforms/**null**/android.jar does not exist

I miss something?

build.gradle:
```
buildscript {
    repositories {
        jcenter()
    	mavenCentral()
    }
    
    dependencies {
        classpath 'org.javafxports:jfxmobile-plugin:1.0.0-b4'
    	classpath 'me.tatarka:gradle-retrolambda:2.5.0'
    }
}

apply plugin: 'org.javafxports.jfxmobile'
apply plugin: 'application'
apply plugin: 'com.android.application'
apply plugin: 'me.tatarka.retrolambda'

mainClassName = 'erico.teste.Main'

repositories {
    jcenter()
    mavenCentral()
}

dependencies {
    testCompile group: 'junit', name: 'junit', version: '4.+'
    compile group: 'com.guigarage', name: 'responsivefx', version: '0.7'
}

jfxmobile {
	ios {
        forceLinkClasses = [ 'erico.teste.**.*' ]
    }
    android {
        applicationPackage = 'erico.teste'
    }
}
```

**Comment by evant** [comment]
This is probably related to https://github.com/evant/gradle-retrolambda/issues/53. Seems like platform jar can be in strange places depending on what you're using. There should probably be another configuration option to define it directly.

**Comment by klutzer** [comment]
Thanks evant.. I take the print below:
![image](https://cloud.githubusercontent.com/assets/10657645/6763700/cd806322-cf67-11e4-8c3f-0138690b3408.png)

**Comment by klutzer** [comment]
Can you give me any idea how to solve this problem? This bug only happens when I add some external library in my project which uses Java 8.

**Comment by evant** [comment]
I just noticed you said "when I add some external library in my project which uses Java 8". This plugin works only on your source code, _not_ your dependencies. All dependencies must be java 6/7 to work on android, though you _could_ try to run them through retrolambda yourself if you want to. I also noticed in your stacktrace that you have "The 'java' plugin has been applied, but it is not compatible with the Android plugins."  Android and java plugins are not compatible, you cannot apply both at the same time, and I don't see why you'd want to.
